.changelogs/issue_1717-wpengine-cache.yml

```
@@ -0,0 +1,6 @@
+significance: patch
+type: fixed
+links:
+  - "#1717"
+entry: Added automatic exclusion of dashboard and checkout pages, including
+  dashboard endpoints like 'lost-pasword', from the WP Engine server-side cache.
```

```suggestion
entry: Added automatic exclusion of "no cache" pages from the WP Engine server-side cache.
```

No cache extends outside of the dashboard since 6.4.0